lib/index.js

```diff
@@ -27,6 +28,7 @@ exports.agents = {
 exports.request = function (method, url, options, callback, _trace) {
 
     options = options || {};
+    var logger = new Logger();
```

Comment:
We should be fine to make logger static.  Maybe we should export the file/console parts so that it can be changed in tests more easily?